src/components-nextjs/Sitemap/types.ts

```diff
@@ -0,0 +1,5 @@
+export type SitemapUrl = {
+    loc: string;
+    changefreq?: string;
+    priority?: string;
+};
```

Comment:
It would be nice to use more descriptive property names like `location` and `changeFrequency`.